types/bagpipes/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "../tslint.json" }
```

Comment:
We've recently updated it; please merge in master and change the value to `dtslint/dt.json`.